php/hyperf/.php_cs

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
could you please remove this file ?

I prefer to have the same conventions on all php frameworks, or at least if you fine it useful to create an issue to adopt community style for all php frameworks